features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/RecordedDataProcessor.kt

```diff
@@ -36,13 +38,20 @@ internal class RecordedDataProcessor(
     override fun processResources(
         item: ResourceRecordedDataQueueItem
     ) {
-        val enrichedResource = EnrichedResource(
-            resource = item.resourceData,
-            applicationId = item.applicationId,
-            filename = item.identifier
-        )
+        val resourceHash = item.identifier
+        val isKnownResource = resourcesDataStoreManager.wasResourcePreviouslySent(resourceHash)
```

Comment:
it is better to use present tense, here is a proposed naming change:

```suggestion
        val isKnownResource = resourcesDataStoreManager.isPreviouslySentResource(resourceHash)
```